src/test/scala/org/apache/mesos/chronos/ChronosSpec.scala

```diff
@@ -0,0 +1,18 @@
+package org.apache.mesos.chronos
+
+import mesosphere.chaos.http.HttpConf
+import org.apache.mesos.chronos.scheduler.config.SchedulerConfiguration
+import org.rogach.scallop.ScallopConf
+import org.specs2.mock.Mockito
+import org.specs2.mutable.SpecificationWithJUnit
+
+trait ChronosSpec extends SpecificationWithJUnit with Mockito {
+  def makeConfig(args: String*): SchedulerConfiguration with HttpConf = {
```

Comment:
I do not particularly like inheritance for getting additional methods in scope. This method could simply be part of a ChronosTestHelper object.